database/data_seeding/nosql_seeder.js

```diff
@@ -2,45 +2,44 @@ const { user, password } = require('../../database_configs/nosql_database.config
 const nano = require('nano')(`http://${user}:${password}@localhost:5984`);
 const { generateData } = require('./data_generator.js');
 
-const destroyDb = async (dbName) => {
-  await nano.db.destroy(dbName);
-}
+const seedNoSqlData = async (numRecords, pace) => {
+
+  if ((numRecords / pace) - Math.floor(numRecords / pace) !== 0) {
+    throw new Error('Number of records must be divisible by pace (ex: 1,000 records at a pace of 100)');
+  }
+
+  const numBatches = numRecords / pace;
+  const copyOfNumRecords = numRecords;
+
+  try {
+    await nano.db.destroy('product_features');
+    await nano.db.create('product_features');
+  } catch (err) {
+    throw new Error('Error initializing database: ', err);
+  }
+
+  const pf = nano.use('product_features');
+
+  let currentBatch = 1;
+  let startingId = 1;
 
-const createDb = async (dbName) => {
-  await nano.db.create(dbName)
+  while (numRecords > 0) {
+    const data = generateData(1000, startingId, 'nosql');
+
+    await pf.bulk({ docs: data })
```

Comment:
You can disregard my previous comment about using this function. You found it and did a pretty big overhaul of this. This looks really good.

Reply:
So weird that the code from my previous commits showed up like that, yeah i was originally just testing inserting one at a time and later switched to bulk to bulk insert